_sass/blog/_layout.scss

```diff
@@ -103,7 +103,58 @@
       margin-right: 10px;
       margin-left: 10px;
     }
+  }
 
+  .dropdown-content {
+    display: none;
+  }
+  @media screen and (min-width: 609px) {
+    .dropdown {
+      position: relative;
+      display: inline-block;
+
+      .dropdown-content {
+        background-color: $header-heading-color;
+        border: 1px solid $grey-color-light;
+        border-top: none;
+        z-index: 1;
+        position: absolute;
+        overflow: hidden;
+        display: block;
+
+        visibility: hidden;
+        transform-origin: top;
+        -webkit-transform-origin: top;
+        transform: scaleY(0);
+        -webkit-transform: scaleY(0);
+        -ms-transform: scaleY(0);
+
+        transition: transform 01.25s ease;
```

Comment:
Shouldn’t this be .25, too? Or 1.25 on the prefixed lines?

Reply:
Oops, that was for testing, it should be 0.25 too. Will fix soon, thanks!